modules/tinymce/src/plugins/accordion/main/ts/ui/Buttons.ts

```diff
@@ -20,6 +14,26 @@ const register = (editor: Editor): void => {
   const onAction = () => editor.execCommand('InsertAccordion');
   editor.ui.registry.addButton('accordion', { icon: 'accordion', text: 'Accordion', onSetup: onSetup(editor), onAction });
   editor.ui.registry.addMenuItem('accordion', { icon: 'accordion', text: 'Accordion', onSetup: onSetup(editor), onAction });
+
+  editor.ui.registry.addToggleButton('accordiontoggle', {
+    text: 'Toggle accordion',
+    tooltip: 'Toggle accordion',
+    onAction: () => editor.execCommand('ToggleAccordion')
+  });
+
+  editor.ui.registry.addToggleButton('accordionremove', {
+    text: 'Remove accordion',
```

Comment:
I noticed in strings for translations that it should be "Delete accordion"
https://github.com/tinymce/tinymce-enterprise-language-packs/blob/main/src/text/pot/core/tinymce_core.pot#L1258

Reply:
Nice point! Updated.